documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1022,6 +1022,19 @@ You can use the {prodname} connector for MongoDB with https://www.mongodb.com/at
 Note that MongoDB Atlas only supports secure connections via SSL, i.e. the xref:mongodb-property-mongodb-ssl-enabled[`+mongodb.ssl.enabled`] connector option _must_ be set to `true`.
 endif::community[]
 
+
+[[mongodb-optimal-oplog-config]]
+=== Optimal Oplog Config
+
+
+The {prodname} MongoDB connector works by obtaining changes from https://www.mongodb.com/docs/manual/core/replica-set-oplog/[Oplog] using https://www.mongodb.com/docs/manual/changeStreams/[change streams]. As oplog is a https://www.mongodb.com/docs/manual/reference/glossary/#std-term-capped-collection/[capped collection], older entries are overwritten once maximum configured size is reached. For MongoDB connector to be able to deal with failures it is important that the oplog retention config is such that the resume point is available upon restart or retries. If the last oplog position is lost, the connector can not start again and a new connector has to be created afresh - See xref:debezium-mongodb-connector-is-stopped-for-a-long-interval[Connector fails after it is stopped for a long interval if snapshot.mode is set to initial].
```

Comment:
Understood. In that case, adding it will create more confusion. 

I think here we can instead extend the previous statement to _"the connector *may* fails to start depending on your `snapshot.mode` config, and you must create a new connector."_

Reply:
Edited.